src/main/java/org/mule/extensions/jms/api/connection/factory/activemq/ActiveMQConnectionFactoryConfiguration.java

```diff
@@ -96,10 +95,8 @@
   private List<String> trustedPackages;
 
   /**
-   * Indicates if any class from any package can be sent and received. Enabling this is unsafe as malicious payload can
-   * exploit the host system.
-   * This property starting versions 5.12.2 and 5.13.0, ActiveMQ enforces users to explicitly whitelist packages that
-   * can be exchanged using ObjectMessages.
+   * Indicates if any class from any package can be sent and received. Enabling this parameter is unsafe as malicious payload can exploit the host system.
+   * ActiveMQ requires you to explicitly allow packages that can be exchanged using ObjectMessages.
```

Comment:
Do we need to keep the version info? Starting with versions 5.12.2 and 5.13.0, ActiveMQ...